lib/yoti/sandbox/sandbox.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'yoti'
+require 'yoti/configuration'
+require 'yoti/http/signed_request'
+
+require_relative 'sandbox_client'
+require_relative 'profile'
+require_relative 'anchor'
+require_relative 'attribute'
+
+require 'openssl'
+require 'net/http'
+require 'date'
+require 'securerandom'
+
+require 'dotenv'
+Dotenv.load
+
+# Singleton for sandbox test resources
+module Sandbox
+  class << self
+    attr_accessor :sandbox_client
+    attr_accessor :dev_key
+    attr_accessor :application
+  end
+
+  def self.setup!
+    read_dev_key!
+    create_application!
+    self.sandbox_client = Client.new(
+      app_id: application['id'],
+      private_key: application['private_key'],
+      base_url: ENV['SANDBOX_BASE_URL'],
+      version_id: ENV['SANDBOX_VERSION_ID']
+    )
+    configure_yoti(
+      app_id: sandbox_client.app_id,
+      pem: sandbox_client.key.to_pem
+    )
+    nil
+  end
+
+  def self.configure_yoti(app_id:, pem:)
+    Yoti.configuration = Yoti::Configuration.new
+    Yoti.configuration.client_sdk_id = app_id
+    Yoti.configuration.key = pem
+    Yoti.configuration.key_file_path = ''
+    Yoti.configuration.api_endpoint = "#{ENV['SANDBOX_BASE_URL']}/v1"
```

Comment:
Is there any reason why we can't configure/build the sandbox client like other SDKs? Then we can pass this base url into the constructor (or a method), rather than use an environment variable

We shouldn't need to hard-code the `/v1` here if it's part of the base URL

Reply:
The Yoti client in the Ruby SDK is a singleton. In other SDKs it's a regular object.

See comment below about the v1.